crates/masonry/.cargo/config.toml

```diff
@@ -1,9 +0,0 @@
-[target.'cfg(all())']
-# TODO - Replace with Cargo.toml config
-rustflags = [
-    # Global lints/warnings.
-    # We do this here instead of in the crate root because we want to apply
-    # these to example and test targets as well.
-    "-Aclippy::single_match",
```

Comment:
Ah, yes. I ran into this one in #205. This would explain why it wasn't an issue before

Reply:
There's still an `allow` in `crates/masonry/src/lib.rs` which can be removed, but will have to fix the code as well at that time. Another PR, probably another day.